starlite/middleware/session/file_backend.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime, timedelta
+from os import PathLike
+from typing import Optional, Tuple, Type
+
+import anyio
+import orjson
+
+from starlite.middleware.session.base import ServerSideBackend, ServerSideSessionConfig
+
+FileStorageMetadataWrapper = Tuple[str, str]
```

Comment:
You can also use a named tuple, will make acces nicer by allowing `.attribute` rather than indexed access.

Reply:
Good idea! Done